ui/src/index.html

```diff
@@ -13,6 +13,49 @@
     <script src="https://apis.google.com/js/platform.js"></script>
     <link href="https://fonts.googleapis.com/css?family=Montserrat:400,600,700" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+
+    <script type="text/javascript">
+      // This data configures the Zendesk web widget with settings to show only
+      // the "contact us" form. See https://developer.zendesk.com/embeddables/docs/widget/
+      // for API docs.
+      window.zESettings = {
+        webWidget: {
+          chat: {
+            suppress: true,
+          },
+          color: {
+            // This is an AoU dark purple color.
+            theme: "#262262"
+          },
+          contactForm: {
+            attachments: true,
+            subject: false,
+            // We include a tag indicating that this support request was filed via the
+            // research workbench. This helps distinguish from tickets filed via other
+            // AoU sub-products, e.g. Research Hub or Data Browser.
+            tags: ['research_workbench'],
+            title: 'Help Desk'
+          },
+          contactOptions: {
+            enabled: true,
+            contactButton: {
+              '*': 'Send'
+            },
+          },
+          helpCenter: {
+            suppress: true,
+          },
+          talk: {
+            suppress: true,
+          },
+        }
+      };
+    </script>
+
+    <!-- Start of aousupporthelp Zendesk Widget script -->
```

Comment:
nit: Is the intention to update this via a script? If not, do we need both these HTML comments?

Reply:
Removed – I'd just left them from Zendesk's copy-paste snippet.